### PR TITLE
Added updated 1.7.6 and 2.0.2.1 and sorted jira numbers in patchews

### PR DESCRIPTION
**Pull request by tarunpx**
Signed-off-by: Tarun Rajput <tarun@Taruns-MacBook-Pro.local>

### CODE REVIEW

File: content/reference/release-notes/_index.md [hunk 1]
```diff
@@ -68,6 +107,31 @@ series: reference
   Workaround: This is a variant of the previous errata. 
   For volume with replication factor set to 1, Cloudsnap backup does not resume after the node with replica goes down.
 
+## 1.7.6
+
+* PWX-7304 - PX keeps restarting, if journal device made read-only
+* PWX-7348 - PX keeps restarting, VM reboot after journal device made “offline”
+* PWX-7453 - btrfs send was not cleaned properly by px after cloudsnap failure of invalid keys
```
Comment (venkatpx) [inline content/reference/release-notes/_index.md]: Please remove reference to btrfs. Just reword this is as "cloudsnap cleanup didn't complete properly in cases where errors were encountered when tranmitting the diffs"

Comment (tarunpx) [reply content/reference/release-notes/_index.md]: Done.

File: content/reference/release-notes/_index.md [hunk 2]
```diff
@@ -68,6 +107,31 @@ series: reference
...
+* PWX-7481 - Shared volume failed to detach
```
[the remark this answers is not in the document]
Comment (tarunpx) [reply content/reference/release-notes/_index.md]: done

File: content/reference/release-notes/_index.md [hunk 3]
```diff
@@ -68,6 +107,31 @@ series: reference
...
+* PWX-7033 - Objectostore endpoint failover not happening
+* PWX-7179 - io1 volume added to KOPS cluster gets displayed as STORAGE_MEDIUM_MAGNETIC
+* PWX-7212 - Cloudsnap-Restore: Increase restore verbose level for error cases
+* PWX-7267 - LIC: Hide unknown/non-handled licenses
+* PWX-7299 - export pool_status as a stat for prometheus
+* PWX-7364 - Namespace stuck volume issue 
+
+## 1.7.4
+
+* PWX-7142 - LIC: SuperMicro misinterpreted as VM [roblox]
```
[the remark this answers is not in the document]
Comment (tarunpx) [reply content/reference/release-notes/_index.md]: done

File: content/reference/release-notes/_index.md [hunk 4]
```diff
@@ -68,6 +107,31 @@ series: reference
   Workaround: This is a variant of the previous errata. 
   For volume with replication factor set to 1, Cloudsnap backup does not resume after the node with replica goes down.
 
+## 1.7.6
+
+* PWX-7304 - PX keeps restarting, if journal device made read-only
+* PWX-7348 - PX keeps restarting, VM reboot after journal device made “offline”
+* PWX-7453 - btrfs send was not cleaned properly by px after cloudsnap failure of invalid keys
+* PWX-7481 - Shared volume failed to detach
+* PWX-7600 - do not reset coordinator when doing repl-move
```
[the remark this answers is not in the document]
Comment (tarunpx) [reply content/reference/release-notes/_index.md]: PWX-7600 summary updated

**Pull request merged**